production/mempool-config.mainnet.json

```diff
@@ -10,6 +10,7 @@
     "POLL_RATE_MS": 1000,
     "INDEXING_BLOCKS_AMOUNT": -1,
     "BLOCKS_SUMMARIES_INDEXING": true,
+    "CPFP_INDEXING": true,
```

Comment:
Please don't modify production configuration settings in this PR, this will be a separate decision later after we get CPFP stable :)